docker/heap-sizing/src/main/java/com/baeldung/docker/heapsizing/PrintXmxXms.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.docker.heapsizing;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.MemoryMXBean;
+import java.lang.management.MemoryPoolMXBean;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class PrintXmxXms {
+
+    private static final Logger logger = Logger.getLogger(PrintXmxXms.class.getName());
+
+    public static void main(String[] args) {
+        logMemory(logger);
```

Comment:
The extracted method isn't really adding much here - let's inline it.

Reply:
It's reused in HeapSizingApplication.java. I wanted to avoid repeating the same code over there.